digits/scheduler.py

```diff
@@ -175,6 +177,30 @@ def add_job(self, job):
             return False
         else:
             self.jobs.append(job)
+
+            if flask._app_ctx_stack.top is not None:
```

Comment:
Does this work?
```python
with app.app_context():
    html = flask.render_template('job_row.html', job = job)
```
Examples:
https://github.com/NVIDIA/DIGITS/blob/v2.2.1/digits/job.py#L161-L162
https://github.com/NVIDIA/DIGITS/blob/v2.2.1/digits/task.py#L100-L105